data/CentOS.yaml

```diff
@@ -1,2 +1,3 @@
 ---
 lldpd::manage_jq: false
+lldpd::repourl: 'CentOS_%{facts.os.release.major}'
```

Comment:
Why not move it to a os.family-layer? Seems the same value for CentOS and VirtuozzoLinux.

Reply:
```
  - name: 'Distribution Name'
    path: '%{facts.os.name}.yaml'

  - name: 'Operating System Family'
    path: '%{facts.os.family}.yaml'
```
`redhat.yaml` should match first for `os.family` thus adding the `lldpd::repourl` to it would set it as default for all RedHat-based distributions.

Reply:
family redhat also matches for fedora and would break this :(

Reply:
But setting it in `fedora.yaml` would overwrite the value from `redhat.yaml`.
- Why do `fedora_25` and `fedora_26` set `repourl`, but not later versions? 
- What about PSBM? It's also using `lldpd::repourl: 'CentOS_CentOS-6'`.

Reply:
PBSM is not listed in metadata.json. I think on Fedora the repo should not be managed since `lldpd` is available in the repos themselves.